src/toml-text_io.ads

```diff
@@ -0,0 +1,16 @@
+with Ada.Text_IO;
+
+package TOML.Text_IO is
```

Comment:
Can you rename this package to `TOML.File_IO`? Although this package is indeed implemented on top of `Ada.Text_IO`, its UTF-8 decoding processes bytes more than characters, so I think this name would make more sense.

Also can you add at the beginning of this file a comment paragraph that explains that this package provides subprograms to read and write TOML documents from files?